processor/src/lib.rs

```diff
@@ -95,9 +96,10 @@ pub struct ChipletsTrace {
 /// inputs.
 pub fn execute(
     program: &Program,
+    stack: StackInputs,
     inputs: &ProgramInputs,
 ) -> Result<ExecutionTrace, ExecutionError> {
```

Comment:
Is the idea to leave passing advice provider into here for the next PR? If so, I'd still probably rename `inputs` into `advice_inputs` and `stack` into `stack_inputs `.

Reply:
Yes, we will refactor that in the next PR (that should be far simpler than this one)